zhaquirks/ikea/twobtnremote.py

```diff
@@ -135,3 +135,45 @@ class IkeaTradfriRemote2Btn(CustomDevice):
             ARGS: [1, 83],
         },
     }
+
+
+class IkeaTradfriRemote2BtnZLL(CustomDevice):
+    """Custom device representing IKEA of Sweden TRADFRI remote control."""
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=49246 device_type=2080
+        # device_version=248
+        # input_clusters=[0, 1, 3, 9, 258, 4096, 64636]
+        # output_clusters=[3, 4, 6, 8, 25, 258, 4096]>
+        MODELS_INFO: IkeaTradfriRemote2Btn.signature[MODELS_INFO].copy(),
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zll.PROFILE_ID,
+                DEVICE_TYPE: zll.DeviceType.CONTROLLER,
+                INPUT_CLUSTERS: IkeaTradfriRemote2Btn.signature[ENDPOINTS][1][
+                    INPUT_CLUSTERS
+                ].copy(),
```

Comment:
Some other things I've noticed with this. I own two of these, both using the ZLL profile. One of these pairs and triggers zha_events properly. The second one is not triggering events. So I thought I would remove the device and pair it again. While the stick is in add device mode and I would click on or off, it would show the following output, yet still not triggering any zha_events. This seems odd to me, but I don't know enough about what the expected output by ZHA/zigpy should be.

```
[0x5cd2:1:0x0006] ZCL request 0x0000: []
[0x5cd2:1:0x0006] No handler for cluster command 0
[0x5cd2:1:0x0006] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=16 command_id=1>
[0x5cd2:1:0x0006] ZCL request 0x0001: []
[0x5cd2:1:0x0006] No handler for cluster command 1
[0x5cd2:1:0x0006] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=17 command_id=1>
```

Reply:
> I changed these lines to this:

Can confirm I have the same switch, and using this branch, with this change in the file finaly did the trick for me!

Here is the descriptor from mine:
```
<SimpleDescriptor endpoint=1 profile=49246 device_type=2080 device_version=207 input_clusters=[0, 1, 3, 9, 258, 4096, 64636] output_clusters=[3, 4, 6, 8, 25, 258, 4096]>
```

Reply:
thx!